site/content/courses/15-071-the-analytics-edge-spring-2017/_index.md

```diff
@@ -28,10 +28,10 @@ menu:
 
 ##### Course Features
 
-* [AV lectures]({{< ref "sections/an-introduction-to-analytics" >}})
-* [Interactive assessments]({{< ref "sections/unit-index" >}})
-* [Lecture notes]({{< ref "sections/lecture-and-recitation-notes" >}})
-* [Assignments]({{< ref "sections/assignments" >}})
+* [AV lectures](an-introduction-to-analytics)
+* [Interactive assessments](unit-index)
+* [Lecture notes](lecture-and-recitation-notes)
+* [Assignments](assignments)
```

Comment:
it seems like the ref for section links are gone - any ideas what's up here?

Reply:
(just confirmed that it makes them stop working)

Reply:
hmm, not sure, let me look into it

Reply:
maybe the PR in ocw-to-hugo needs to be rebased to have the previous link fixes in it?

Reply:
I just pushed up some fixes.  Course features in the master json are currently fully qualified URL's to the OCW live site, so the url has to be deconstructed and matched with a course section to create the ref shortcode.  This change probably doesn't belong in this PR honestly, but I figure we should just get in in there because there is probably a better way to construct course features.  Peter was saying that course features on any given course are checkboxes that are checked off in Plone by content managers.  I believe this is an item in Trello, so we can re-address this in a more sane manner later.

Reply:
I should also clarify that when you had checked it out earlier, it seems as if the only refs that were broken were the ones under course features on each course home page, and that would make sense given what I mentioned above.